### ISSUE TITLE
HttpModule registerAsync not working

### ISSUE THREAD

**Issue by kbrustol** [nestjs/nest#1523]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

HttpModule seems to ignore options into useFactory (timeout in my case)

## Minimal reproduction of the problem with instructions

```
@Module({
  imports: [
    HttpModule.registerAsync({
      useFactory: (configService: ConfigService) => ({
        timeout: configService.TIMEOUT,
      }),
      inject: [ConfigService],
    }),
  ],
  providers: [StartSessionService],
})
export class SessionModule {}
```
````
@Injectable()
export class StartSessionService {
  constructor(private readonly httpService: HttpService) {}

  async execute(): Promise<void> {
    const session = { ... };

    await this.httpService
      .post('session', session)
      .toPromise()
      .catch(error => {
        if (error.response) {
          throw new HttpException(
            HttpStatus[error.response.status],
            error.response.status,
          );
        } else {
          const httpCode = 503;
          throw new HttpException(HttpStatus[httpCode], httpCode);
        }
      });
  }
}
````
## Environment

<pre><code>
Nest version: 5.7.1

 
For Tooling issues:
- Node version: 11.7  
- Platform:  Windows 10

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
It was fixed in `5.6.2` (`@nestjs/common`) as far as I remember. Could you update your packages?

**Comment by kbrustol** [comment]
It works, thank you
